src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -234,6 +254,27 @@ function PolicyAccountingPage({policy, connectionSyncProgress}: PolicyAccounting
                       },
                   ]
                 : []),
+            ...(policyConnectedToNetSuite && !shouldShowSynchronizationError
+                ? [
+                      {
+                          description: translate('workspace.netsuite.subsidiary'),
+                          iconRight: Expensicons.ArrowRight,
+                          title: netSuiteSelectedSubsidiary,
+                          wrapperStyle: [styles.sectionMenuItemTopDescription],
+                          titleStyle: styles.fontWeightNormal,
+                          shouldShowRightIcon: netSuiteSubsidiaryList.length > 1,
+                          shouldShowDescriptionOnTop: true,
+                          pendingAction: policy?.connections?.netsuite?.options?.config?.pendingFields?.subsidiary,
+                          brickRoadIndicator: policy?.connections?.netsuite?.options?.config?.errorFields?.subsidiary ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+                          onPress: () => {
+                              if (!(netSuiteSubsidiaryList.length > 1)) {
+                                  return;
+                              }
+                              Navigation.navigate(ROUTES.POLICY_ACCOUNTING_NETSUITE_SUBSIDIARY_SELECTOR.getRoute(policyID));
+                          },
+                      },
+                  ]
+                : []),
```

Comment:
Looks like this part of the logic led to the following issue:
- https://github.com/Expensify/App/issues/48274#issuecomment-2368480825

Which we solved by returning empty object when there's no value for subsidiary, more details in issue [proposal](https://github.com/Expensify/App/issues/48274#issuecomment-2318192111).